src/OrchardCore/OrchardCore.Abstractions/Caching/ISignal.cs

```diff
@@ -16,7 +16,9 @@ public static class SignalExtensions
         /// <summary>
         /// Adds a Signal (if not already present) to be sent at the end of the shell scope.
         /// </summary>
+#pragma warning disable IDE0060 // Remove unused parameter
```

Comment:
Comment suppressions so it's apparent why there are there.

Reply:
No problem, I can remove these warnings by using the discard symbol `_`

So I will remove all these pragmas ;)